src/api.ts

```diff
@@ -61,6 +62,17 @@ export async function addDependency(
 
   const names = Array.isArray(name) ? name : [name];
 
+  if (resolvedOptions.packageManager.name === "deno") {
+    for (let i = 0; i < names.length; i++) {
+      if (!/^.*?:.+$/.test(names[i])) {
+        console.info(
+          `No prefix found for ${names[i]}. Defaulting to "npm:${names[i]}".`,
+        );
```

Comment:
I think we can also do it silently since nypm in general does not requires `npm:` prefix today